### DISCUSSION TITLE
File upload settings not saved per user

### DISCUSSION THREAD

**Discussion by MalteMagnussen** [Forceu/Gokapi#194] [General]
When uploading a file, you can select Expiry, number of downloads, and password. 

These seems to be saved whenever you upload a file. So far so good. 

However, now that we are getting many users on our Gokapi instance, we have discovered that the three settings seems to be saved globally!  
So whenever someone else changes them, they're changed for everyone. 

We would have assumed this to be saved per user. And to be stored in something like local cookies, as opposed to the SQLite DB (I assume that is where it is stored).

**Comment by Forceu** [comment]
Thank you for the feedback. That is indeed currently the case, but I assume it is a good idea to save the defaults as a cookie instead. That way they are user-independent. Tracked in #196

Edit: Since v2.0 the settings are stored locally
